src/test/java/teammates/test/cases/logic/InstructorsLogicTest.java

```diff
@@ -52,6 +53,7 @@ public void testAll() throws Exception {
         testDeleteInstructor();
         testDeleteInstructorsForGoogleId();
         testDeleteInstructorsForCourse();
+        testGetCoOwnersForCourse();
```

Comment:
You should test this method before `testDeleteInstructorsForCourse()` since the instructors for that course are all deleted in that test 😂